src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/net/SubscriptionManagerTest.java

```diff
@@ -2,16 +2,19 @@
 
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.SubscribeResponse;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.net.GRPCConnectionManager;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.util.InstanceDetails;
 import com.google.common.collect.Sets;
 import java.util.Collections;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 
-
+// TODO: remove
```

Comment:
It's ok to remove this @Ignore and change grpcConnectionManager = new GRPCConnectionManager(false). TLS semantics are tested elsewhere.

Reply:
I added the change for it to get the right files.